fix(utility): nullable formatting

georgettica opened the pull request:
same as #13793 

related to #13791

georgettica committed:
fix(utility): nullable formatting

iSazonov commented on src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs:
```diff
@@ -50,7 +50,7 @@ public SwitchParameter AutoSize
             {
                 if (_autosize.HasValue)
-                    return _autosize.Value;
+                    return _autosize;
```
We can not assign nullable type to non-nullable one.
Below too.
```suggestion
                    return _autosize.GetValueOrDefault();
```

iSazonov replied:
You should do the same for rest of changes.

georgettica committed:
fix: change return value

As this is done on a machine with no formatting, I need to rely on the developer on the other end to verify the code

Co-authored-by: Ilya <darpa@yandex.ru>

georgettica commented:
Conversion issues here aswell, will be fixed tomorrow hopefully

georgettica commented:
cancelling as I think someone else is doing these
